portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,13 +63,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String username = request.getParameter("username");
     String message = request.getParameter("comment-or-question");
     String maxNumCommentsString = request.getParameter("max-num-comments");
+    int maxNumComments;
     try {
       maxNumComments = Integer.parseInt(maxNumCommentsString);
     } catch (NumberFormatException e) {
-      System.err.println("Could not convert to int: " + maxNumCommentsString);
-      System.err.println("Using default value of " + defaultMaxNumComments + ".");
-      maxNumComments = defaultMaxNumComments;
+      System.err.format("Could not convert to int: %d%n", maxNumCommentsString);
```

Comment:
there's a mismatch here since `maxNumCommentsString` is a string. s/%d/%s/.

Reply:
Ok, I'll do that replacement.